04/app.js

```diff
@@ -6,4 +6,18 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const day = new Date().getDate();
+const month = new Date().getMonth() + 1;
+const year = new Date().getFullYear();
+
+function isUserBirthday() {
+
+    if(parseInt((user.born.day) === parseInt(day)) && parseInt(user.born.month) === parseInt(month)) {
```

Comment:
W przypadku `day` oraz `month` nie trzeba robić `parseInt()` bo to już jest liczba :)